src/components/AppMapSettings.ts

```diff
@@ -18,14 +18,16 @@ export default class AppMapSettings extends Vue {
   s: Settings | null = null;
 
   optionsMapType = Object.freeze([
-    { value: 'Totk', text: 'Sky, Surface and Depths' },
+    { value: 'Any', text: 'Sky, Surface and Depths' },
     { value: 'SmallDungeon', text: 'Shrines (SmallDungeon)' },
     { value: 'LargeDungeon', text: 'Temples (LargeDungeon)' },
     { value: 'NormalStage', text: 'Special Maps (NormalStage)' },
+    { value: 'MainField', text: 'Surface, Sky, Caves and Chasms' },
```

Comment:
I think the description should just be "Sky and Surface" or something like "Overworld (MainField)". Otherwise, people will be left wondering if the Any option also includes caves/chasms.

Reply:
Sky and Surface (MainField)
Done